recipe_scrapers/_utils.py

```diff
@@ -65,21 +73,35 @@
 
 def _extract_fractional(input_string: str) -> float:
     input_string = input_string.strip()
-    if "/" in input_string:
-        # for example "1 1/2" is matched
-        components = input_string.split(" ")
-        whole_part, fractional_part = components[0], components[-1]
+
+    # Handling mixed numbers with unicode fractions e.g., '1⅔'
+    for unicode_fraction, value in FRACTIONS.items():
+        if unicode_fraction in input_string:
+            parts = input_string.partition(unicode_fraction)
+            whole_number_part = parts[0]
+            fraction_part = value
+
+            whole_number = float(whole_number_part) if whole_number_part else 0
```

Comment:
It's a nitpick, but to avoid repeating the variable name, achieving some line length reduction, I'd suggest using an `or` statement to handle default-if-var-empty here:

```suggestion
            whole_number = float(whole_number_part or 0)
```

(strictly speaking it's mixing a `str`/`int` type within there -- but it's ok, `float(...)` can handle both)